### ISSUE TITLE
Retriggerable LFO : feature request

### ISSUE THREAD

**Issue by muziker** [zynaddsubfx/zyn-fusion-issues#263]
Is it possible to make all envelopes retriggerable on new midi note input? As it stands, without a retrig it is impossible to do good 808 style kick

**Comment by muziker** [comment]
As in the frequency LFOs are not retriggerable, making it impossible to do 808 basses

**Comment by fundamental** [comment]
Interesting. I've had plenty of desire for work into the area of how envelopes can retrigger (design is pretty standard sticking with classic ADSR setups, but not 100% nailed down in the custom envelope case), but I haven't seen or thought about LFO triggering. Seems like functionality which would be in-scope, but not currently done.

**Comment by muziker** [comment]
Yes the newer synths like helm and surge have custom routable matrices and retriggering capabilities. This is a nice video about something of that sort 

https://youtu.be/RSB_eurnokQ

**Comment by muziker** [comment]
Is there a zynaddsubfx internal design document for the code somewhere?
